ROMFS/px4fmu_common/init.d/rc.vtol_apps

```diff
@@ -12,30 +12,30 @@
 # INAV (deprecated)
 if param compare SYS_MC_EST_GROUP 0
 then
-	echo "ERROR [init] Estimator INAV deprecated. Using EKF2"
-	param set SYS_MC_EST_GROUP 2
-	param save
+		echo "ERROR [init] Estimator INAV deprecated. Using EKF2"
```

Comment:
one tab too much

Reply:
THanks @bkueng ,

That happened in one other instance as well.  I've pushed up that fix and removed a few extra newlines at the end of files as well.

In case it is useful, I initially created this PR without the tab/whitespace changes, but the flash size was too big until I did the tab/whitespace work.

Thanks again!

-Mark

Reply:
Ah good. It's also a problem in `rcS` - the diff will be considerably smaller.

> In case it is useful, I initially created this PR without the tab/whitespace changes, but the flash size was too big until I did the tab/whitespace work.

That is strange since you *add* whitespace. Also the ROMFS pruner script should remove whitespace, so that it does not make a difference in terms of Flash usage.